docs/endpoints/campaigns.md

```diff
@@ -72,3 +72,26 @@ Example Response:
     }
 }
 ```
+
+## Update A Specific Campaign
+
+```
+PUT /api/v3/campaigns/:campaign_id
+```
+
```

Comment:
Let's document the body content here (for example, how we document [the `status` and `comment` fields here](https://github.com/DoSomething/rogue/blob/master/docs/endpoints/reviews.md#reviews)).